Files/Views/Pages/Properties.xaml

```diff
@@ -143,7 +143,15 @@
                     Grid.Column="1"
                     Margin="20,0,0,0"
                     IsTextSelectionEnabled="True"
-                    Text="{x:Bind ItemProperties.ItemSize, Mode=OneWay}" />
+                    Text="{x:Bind ItemProperties.ItemSize, Mode=OneWay}"
+                    Visibility="{x:Bind ItemProperties.ItemSizeVisibility, Mode=OneWay}" />
+                <ProgressBar
```

Comment:
Add namespace `xmlns:muxc="using:Microsoft.UI.Xaml.Controls"` to the root element of this document and change all ProgressBars in this document to `muxc:ProgressBar`. In `GetHashForFile` method change `ProgressBar` to `Microsoft.UI.Xaml.Controls.ProgressBar`.

Reply:
Done